src/frontend/apps/lti_site/components/ErrorComponents/BoundaryScreenError/index.spec.tsx

```diff
@@ -0,0 +1,68 @@
+import render from 'utils/tests/render';
+import React, { Fragment, useEffect } from 'react';
+import { screen } from '@testing-library/react';
+import { BoundaryScreenError } from 'components/ErrorComponents';
+import { ErrorBoundary } from 'react-error-boundary';
+
+jest.mock('data/stores/useAppConfig', () => ({
+  useAppConfig: () => ({
+    static: {
+      img: {
+        marshaWhiteLogo: 'path/to/image.png',
+        errorMain: 'path/to/image.png',
+      },
+    },
+  }),
+}));
+
+const ErrorComponent = ({ withError }: { withError: boolean }) => {
+  useEffect(() => {
+    if (withError) {
+      throw new Error('Error generated by ErrorComponent');
+    }
+  }, []);
+
+  return <Fragment>No Error</Fragment>;
+};
+
+describe('<BoundaryScreenError />', () => {
+  it('child component rendered without error', () => {
+    render(
+      <ErrorBoundary
+        fallbackRender={({ error }) => (
+          <BoundaryScreenError code={500} message={error.message} />
+        )}
+      >
+        <ErrorComponent withError={false} />
+      </ErrorBoundary>,
+    );
+
+    expect(screen.getByText('No Error')).toBeInTheDocument();
```

Comment:
```suggestion
    screen.getByText('No Error');
```

`GetByText` throws an error if the text is not in the document.

Reply:
Based on that:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-get-variants-as-assertions

Reply:
:+1: I didn't remember this one